packages/perspective/README.md

```diff
@@ -31,6 +31,8 @@ For more information, see the
         * [.remove_delete(callback)](#module_perspective..view+remove_delete)
     * [~table](#module_perspective..table)
         * [new table()](#new_module_perspective..table_new)
+        * [.get_index()](#module_perspective..table+get_index)
+        * [.get_limit()](#module_perspective..table+get_limit)
```

Comment:
Though I know I have not done this consistently in the past, I will make sure the docs are disted as part of the release cycle so we do not need these inter-mingled in our everyday review process.